### ISSUE TITLE
[bug]: Image generation fails with a server error when starting InvokeAI offline.

### ISSUE THREAD

**Issue by StellarBeing25** [invoke-ai/InvokeAI#6623]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### Operating system

Windows

### GPU vendor

Nvidia (CUDA)

### GPU model

GTX 1660

### GPU VRAM

6GB

### Version number

4.2.6

### Browser

Firefox

### What happened

![SC](https://github.com/user-attachments/assets/8e68e765-f4e0-4056-9f9f-5a87f0b5202c)

Image generation fails with a server error when starting InvokeAI offline.
Image generation starts only when connected to the internet for the first generation however subsequent generations can work offline until switching models.
The error reappears when switching models while offline. 

I am using SD 1.5 models in safetensor format without converting them to diffusers.

### What you expected to happen

InvokeAI should work offline.

**Comment by psychedelicious** [comment]
With single-file (checkpoint) loading, diffusers still needs access to the models' configuration files. Previously, when we converted models, we used a local copy of these config files. With single-file loading, we are no longer referencing the local config files so diffusers is downloading them.

The latest diffusers release revises the single-file loading logic. I think we'll need to upgrade to the latest version, then review the new API to see what our options are.

**Comment by someaccount1234** [comment]
This makes it completely ONLINE ONLY!
The configs folder is right there local, ready to be used lol!
Wasted a good hour+ trying to fix it.
Please fix this, unusable until then!

**Comment by psychedelicious** [comment]
@lstein Forgot to tag you - I think we should be able to fix this up pretty easily.

**Comment by someaccount1234** [comment]
Still the same error! 
Cannot use offline at all!

InvokeAI demands internet connection to download config files that are already local! Every time you change the model!

**Comment by jameswan** [comment]
Server Error (3)
RuntimeError: Parent directory

is an error when you attempt to run two instances Invoke on different ports and attempt to use both GPUs. This is a new error as multiple GPUs was working in version 2 of InvokeAI

**Comment by psychedelicious** [comment]
@jameswan That's an entirely different problem. Please create your own issue. Note that Invoke v2 is _ancient_.

**Comment by psychedelicious** [comment]
@someaccount1234 Yes, this is still a problem. We will close this issue when it is resolved.

**Comment by TobiasReich** [comment]
If some more StackTrace is needed, I provided it in my (duplicated) issue: 
https://github.com/invoke-ai/InvokeAI/issues/6702

**Comment by psychedelicious** [comment]
Thanks @TobiasReich I saw that.

This isn't a mysterious issue, the cause is very clear.

I experimented the other day with providing the config files we already have on disc but `diffusers` couldn't load the tokenizer or text encoder. It's not obvious to me why. 

It doesn't matter anyways, though, because `diffusers` just refactored the API we are using to load models. Whatever issue I'm running into may well no longer exist. So we need to update the `diffusers` dependency (one of our core deps), adapt to some other changes they have made, and then figure out how to provide the config files required.

**Comment by MOzhi327** [comment]
Now, in the infinite canvas, every time a new image is uploaded, an internet connection is required. Maybe it's time to go back to the old version.

**Comment by psychedelicious** [comment]
@MOzhi327 No, that's not how it works. There's no internet connection needed when using canvas. What makes you think an internet connection is required?

**Comment by MOzhi327** [comment]
@psychedelicious Thank you for the reply. On my side, if the VPN is turned off, there is no way to load the model, as follows. 
![tmp8EC1](https://github.com/user-attachments/assets/7a42bf61-e678-4ce7-b425-fbdd37864246)
When I turned on the VPN and generated the image, I could continue to generate without relying on the VPN. Today, I once turned off the VPN. Without adjusting any parameters, when generating the image, a prompt of network connection failure appeared again. (I just tested it again. I can still continue to generate after turning off the VPN. Maybe it was caused by other reasons before. Sorry.) 
Anyway, every time the model is loaded, an internet connection is required. This is very inconvenient for me. Because using a VPN will cause problems with the use of my other software, so I temporarily consider using the old version first.

**Comment by psychedelicious** [comment]
@MOzhi327 Ok, thanks for clarifying. Yes, we know about the internet connectivity issue and will fix it.

**Comment by MOzhi327** [comment]
@psychedelicious Thank you very much

**Comment by webtv123** [comment]
this problem has been going on for a long time to fix it so it runs off line and they keep upgrading and ignoring the issue, word is getting around that this maybe spyware because if it worked before then why does it no work  now, Somebody had to intentionally make it run online only.

**Comment by psychedelicious** [comment]
The problem was introduced when we implemented single-file loading in v4.2.6 on 15 July 2024. We have a few large projects that are taking all contributors' time and which are both resource and technical blockers to resolving this issue.

You do not need to use single file loading in the first place. You can convert your checkpoint/safetensors models to diffusers before going offline (button in the model manager) and then there's no internet connection needed to generate.

**Comment by webtv123** [comment]
> The problem was introduced when we implemented single-file loading in v4.2.6 on 15 July 2024. We have a few large projects that are taking all contributors' time and which are both resource and technical blockers to resolving this issue.
> 
> You do not need to use single file loading in the first place. You can convert your checkpoint/safetensors models to diffusers before going offline (button in the model manager) and then there's no internet connection needed to generate.

THANK YOU.

**Comment by Asherathe** [comment]
As of today, I've suddenly begun having this issue due to an unspecified SSL error with HuggingFace, even though I have a working internet connection. I had no idea internet access was required when using safetensor format. I really wish this was communicated more clearly.

**Comment by Rin247** [comment]
This is till an issue with v6.13.6.
